entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+python3 generate_config.py --certbot && certbot -n certonly; crond && ./generate_config.py --postfix && postfix start-fg
+
+# Idea taken from https://github.com/Mailu/Mailu/blob/master/core/postfix/Dockerfile
+HEALTHCHECK --start-period=350s CMD echo QUIT|nc localhost 25|grep "220 .* ESMTP Postfix"
```

Comment:
should this HEALTHCHECK be moved into the Dockerfile?

Reply:
I thought it served the same purpose when starting the container?

Reply:
ah `HEALTHCHECK` is only used inside a Dockerfile and it isn't recognized in a bash script.